### PR TITLE
The stake tab fully functional

### LINKED ISSUE #197: Staking - Tabs and Stake Tab
blocked by #185 

## Overview
- As a user, I want to view the current exchange rate between TLOS and sTLOS
- As a user I want to see an explanation on why I should stake TLOS
- As a user I want to be able to stake any amount up to the maximum I can stake of TLOS
- As a user I want to click the amount of TLOS that I have and have that input the maximum amount I can stake on the stake input control
## Acceptance Criteria
- Tabs
  - On Stake view, Display Tabs for Stake, Unstake and Withdraw Actions
  - Stake Tab
    -  Display current exchange rate between TLOS and sTLOS
    - Input for Stake Telos with 0 TLOS placeholder, with display of TLOS available that is clickable and fills input with the max amount stakeable
    - Receive STELO input displays amount of sTLOS to be received based on Stake TLOS input, but also takes inputs and changes Stake Telos input to match the amount of sTLOS entered
    - Stake call to action button that executes staking action
    - Why Stake Telos information area
    - Mobile view should not include the Why Stake Telos information area on the tab

### PR DESCRIPTION
**Pull request by Viterbo**
# DUPLICATE SEE #595 FOR DEPLOY PREVIEW

# Fixes #197

## Description
This PR implements the first of the three tabs for the Staking Page. It is rebased with branch 185-staking-top-section-fully-functional which currently has an open PR for the TopHeader for this same page.

## Test scenarios
You can run this PR locally by executing the following terminal commands:
`git fetch --all; git checkout 197-staking-tabs-and-stake-tab; yarn; yarn dev`
- Log in and go to http://localhost:8081/evm/staking?tab=stake
  - You should see the screen displaying the staking form looking similar to the screenshot below.
  - Both inputs should display up to two decimals.
  - Staking should work as expected:
    - You should get two notifications: 
      - Neutral: "Staking <amount> TLOS"
      - Success: Confirmed with explore trx button
    - Your staking balance in the top header should increment

## Screenshots

Captura desde 2023-09-03 13-57-56

## Checklist:

-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [x] I have cleaned up the code in the areas my change touches
-   [x] My changes generate no new warnings
-   [x] Any dependent changes have been merged and published in downstream modules
-   [x] I have checked my code and corrected any misspellings
-   [x] I have removed any unnecessary console messages
-   [x] I have included all english text to the translation file and/or created a new issue with the required translations for the currently supported languages
-   [x] I have tested for mobile functionality and responsiveness
-   [x] I have added appropriate test coverage

### CODE REVIEW

File: src/antelope/stores/utils/trx-utils.ts [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+export async function subscribeForTransactionReceipt(account: AccountModel, response: TransactionResponse): Promise<ethers.providers.TransactionReceipt> {
+    if (account.isNative) {
+        throw new AntelopeError('Not implemented yet for native');
+    } else {
+        const authenticator = account.authenticator as EVMAuthenticator;
+        const provider = await authenticator.web3Provider();
+        if (provider) {
+            const whenConfirmed = provider.waitForTransaction(response.hash);
+            // TODO: remove this console error
+            console.error('subscribeForTransactionReceipt CONFIRMED!!', response.hash);
```
Comment (donnyquixotic) [inline src/antelope/stores/utils/trx-utils.ts:17]: Do you want to remove this?

File: src/pages/evm/staking/StakingPageHeader.vue [hunk 2]
```diff
@@ -1,107 +1,229 @@
...
-    secondaryText: prettyPrintToken(tvlAmountBn.value, systemTokenSymbol.value),
+    tooltip: $t('evm_stake.tvl_card_tooltip', { evmNetworkName: evmNetworkName.value, stakedSymbol: stakedToken.symbol, systemSymbol: systemToken.symbol }),
+    secondaryText: prettyPrintToken(tvlAmountBn.value, systemToken.symbol),
     lowContrastSecondaryText: false,
     useSmallBox: true,
 }]);
 
+// creatmos un interval que cada 5 segundos verifique si cada uno de los loadings es true o false.
+// por cada loading que encontremos en true, debemos llamar a actualizar las dependendicas de ese loading
+// si todos los loadings son false, entonces podemos parar el intervalo
```
Comment (donnyquixotic) [inline src/pages/evm/staking/StakingPageHeader.vue:183]: ¿podría traducir al inglés? gracias.

**Review by donnyquixotic — CHANGES_REQUESTED** [review]

File: src/antelope/stores/utils/trx-utils.ts (hunk 1 above)
Comment (Viterbo) [reply src/antelope/stores/utils/trx-utils.ts:17]: :see_no_evil: forgot to remove it

File: src/pages/evm/staking/StakingPageHeader.vue (hunk 2 above)
Comment (Viterbo) [reply src/pages/evm/staking/StakingPageHeader.vue]: That also was supposed to be removed.
fixed!

**Comment by ezra-guzman** [comment]
awesome work man! only one feedback, which is that after successfully staking, I do not see a 'successful' notification after it completes when using walletconnect (ios, desktop with QR, and android)

**Review by donnyquixotic — APPROVED** [review]

**Comment by donnyquixotic** [comment]
migrated to #595 for deploy preview

**Pull request closed without merging**